manifests/config.pp

```diff
@@ -63,15 +63,17 @@
   contain foreman_proxy::module::httpboot
 
   contain foreman_proxy::module::puppet
+  foreman_proxy::provider { 'puppet_proxy_puppet_api':
+  }
   foreman_proxy::provider { [
-      'puppet_proxy_customrun',
-      'puppet_proxy_mcollective',
-      'puppet_proxy_puppet_api',
-      'puppet_proxy_salt',
-      'puppet_proxy_ssh',
+    'puppet_proxy_legacy',
```

Comment:
Please look at the indenting here. The linting plugin yells at this. There's a reason it was indented the way it was indented.